no_sensor/dags/gcp_custom_ops/gce_commands.py

```diff
@@ -30,22 +30,24 @@ class StopInstanceOperator(BaseOperator):
 
   @apply_defaults
   def __init__(self, project, zone, instance, *args, **kwargs):
-    self.compute = self.get_compute_api_client()
+    self._compute = None
     self.project = project
     self.zone = zone
     self.instance = instance
     super(StopInstanceOperator, self).__init__(*args, **kwargs)
 
   def get_compute_api_client(self):
-    credentials = GoogleCredentials.get_application_default()
-    return googleapiclient.discovery.build(
+    if self._compute is None:
+      credentials = GoogleCredentials.get_application_default()
+      self._compute = googleapiclient.discovery.build(
         'compute', 'v1', cache_discovery=False, credentials=credentials)
+      return self._compute
```

Comment:
The return statements seem within the IF scope because of their indentation.
Should they be outside of the IF so you get a value even if self._compute is not None?

Reply:
Return statements fixed